client/blocks/plan-storage/block.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import filesize from 'filesize';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from 'components/progress-bar';
+
+const ALERT_PERCENT = 80;
+const WARN_PERCENT = 60;
+
+class PlanStorageBlock extends Component {
```

Comment:
Ah got it, though I'm not sure what to call it. Let's avoid block since it's an overloaded term.

PlanStorageCard maybe @iamtakashi ?